mps_youtube/util.py

```diff
@@ -24,6 +24,26 @@
 
 XYTuple = collections.namedtuple('XYTuple', 'width height max_results')
 
+COMMANDS = ['play', 'set', 'album', 'all', 'playurl', 'browserplay',
```

Comment:
This is awkward... I don't like the redundancy, but the current regex method makes it difficult to do anything else...

Reply:
I don't think there's an other way to do it.